Parser/OperatingSystem.php

```diff
@@ -39,7 +39,6 @@ class OperatingSystem extends AbstractParser
         'AIX' => 'AIX',
         'AND' => 'Android',
         'AMG' => 'AmigaOS',
-        'ATV' => 'Apple TV',
```

Comment:
We shouldn't remove anything from this list. If someone has stored that short code and tries to look up the name it will fail afterwards. Might be better to keep using the short code, but give it a new name.

Reply:
I've added back Apple TV, but it fails, because there are no test user agents for this OS anymore.

Reply:
@liviuconcioiu are all Apple TVs running tvOS? If so, we should not add a new OS, but reuse the `ATV` and change the name to `tvOS`

Reply:
The 1st generation came with a OS based on Mac OS X 10.4 Tiger. 2nd, 3rd were based on iOS. After 4th there are based on tvOS.